### ISSUE TITLE
What is highest value for upscale ratio?

### ISSUE THREAD

**Issue by BanikSampad** [xinntao/Real-ESRGAN#847]
I wanted to use the value of upscale ratio is at least 40. Can I use it?

**Comment by JossCamp** [comment]
Yes, you can train a x40 model without problems, really the only limit is your computing power.
